### DISCUSSION TITLE
Critical error each time after 'composer update'

### DISCUSSION THREAD

**Discussion by lxregistry** [symfony/symfony#50854] [Q&A]
![error_composer](https://github.com/symfony/symfony/assets/32699348/d60db720-1e49-48aa-8dc8-b6213622a72f)
After composer update I receive this error:
 CRITICAL  [php] Uncaught Error: Symfony\Component\Translation\Loader\PhpFileLoader::loadResource():
  Return value must be of type array, Symfony\Component\Translation\MessageCatalogue returned ["exception" => TypeError { …}]
!!
!!  In PhpFileLoader.php line 30:
!!
!!    Symfony\Component\Translation\Loader\PhpFileLoader::loadResource(): Return
!!    value must be of type array, Symfony\Component\Translation\MessageCatalogue
!!     returned

this error appear after upgrade to 6.3, the same project running in 6.2 without this error
is there a problem with my code somewhere? any suggestion please

**Comment by MatTheCat** [comment]
Could you provide a stacktrace and share which `$resource` returns a `MessageCatalogue`?
